tests/features/variants.feature

```diff
@@ -91,3 +92,45 @@ Feature: Paragraph types show different fields based on variants.
     When I select "Date" from "Variant"
     And I wait for AJAX to finish
     Then the following fields should be present "Layout, Variant, Link, Title, Description, Meta, URL, Link text, Day, Month, Year"
+
+  # This scenario covers the bug discovered in OPENEUROPA-1843.
+  @javascript @run
+  Scenario: Paragraph variant is correctly saved and persisted when re-saving the content.
+    Given I am logged in as a user with the "Editor" role
+    And I go to "the content management page"
+    And I click "Add content"
+    And I fill in "Title" with "Test page"
+    And I press "List additional actions"
+    And I press "Add Listing item block"
+    And I wait for AJAX to finish
+    And I select "One column" from "Layout" in the 1st "Listing item block" paragraph
+    And I fill in "Title" with "Block title" in the 1st "Listing item block" paragraph
+    And I select "Highlight" from "Variant" in the 1st "Listing item"
```

Comment:
Any reasons for which you don't use the `paragraph` ending to each of these steps?